operations/mimir-mixin/docs/playbooks.md

```diff
@@ -1130,6 +1130,133 @@ The limit protects the system’s stability from potential abuse or mistakes, an
+You can configure the limit on a per-tenant basis by using the `-ingester.max-global-series-per-user` option (or `max_global_series_per_user` in the runtime configuration).
+
+How to **fix**:
+
+- Ensure the actual number of series written by the affected tenant is legit.
+- Consider increasing the per-tenant limit by using the `-ingester.max-global-series-per-user` option (or `max_global_series_per_user` in the runtime configuration).
+
+### err-mimir-max-series-per-metric
+
+This error occurs when the number of in-memory series for a given tenant and metric name exceeds the configured limit.
+
+The limit is primarily used to protect a tenant from potential mistakes on their metrics instrumentation.
+For example, if an instrumented application exposes a metric with a label value including very dynamic data (e.g. a timestamp) the ingestion of that metric would quickly lead to hit the per-tenant series limit, causing other metrics to be rejected too.
+This limit introduces a cap on the maximum number of series each metric name can have, rejecting exceeding series only for that metric name, before the per-tenant series limit is reached.
+You can configure the limit on a per-tenant basis by using the `-ingester.max-global-series-per-metric` option (or `max_global_series_per_metric` in the runtime configuration).
+
+How to **fix**:
+
+- Check the details in the error message to find out which is the affected metric name.
+- Investigate if the high number of series exposed for the affected metric name are legit.
+- Consider reducing the cardinality of the affected metric, by tuning or removing some of its labels.
+- Consider increasing the per-tenant limit by using the `-ingester.max-global-series-per-metric` option.
+- Consider excluding specific metric names from this limit's check by using the `-ingester.ignore-series-limit-for-metric-names` option (or `max_global_series_per_metric` in the runtime configuration).
+
+### err-mimir-max-metadata-per-user
+
+This non-critical error occurs when the number of in-memory metrics with metadata for a given tenant exceeds the configured limit.
+
+Metric metadata is a set of information attached to a metric name, like its unit (e.g. counter) and description.
+Metric metadata can be included by the sender in the write request, and it's returned when querying the `/api/v1/metadata` API endpoint.
+
+Mimir has a per-tenant limit of the number of metric names that have metadata attached.
+This limit is used to protect the whole system’s stability from potential abuse or mistakes.
+You can configure the limit on a per-tenant basis by using the `-ingester.max-global-series-per-user` option (or `max_global_metadata_per_user` in the runtime configuration).
+
+How to **fix**:
+
+- Check the current number of metric names for the affected tenant, running the instant query `count(count by(__name__) ({__name__=~".+"}))`
```

Comment:
Can we suggest the cardinality endpoints here? I think they should give more accurate data because they span the whole head, not only a count at the time of the instant query.

Reply:
Good idea! Will do. Once runbooks will be moved to doc (part of my plan) we'll also be able to link the rest of the doc from here.